j2k/testData/fileOrElement/function/nativeMethods.java

```diff
@@ -0,0 +1,6 @@
+public class Foo {
+    private native final void nativeMethod()
```

Comment:
Please avoid irrelevant syntax errors in testdata. Java does require semicolons :)